internal/apps/cmp/steve/schemas.go

```diff
@@ -80,8 +82,12 @@ func Register(ctx context.Context,
 		ssar:    ssar,
 	}
 
-	apiService.OnChange(ctx, "schema", h.OnChangeAPIService)
-	crd.OnChange(ctx, "schema", h.OnChangeCRD)
+	go func() {
+		loop.New(loop.WithContext(ctx), loop.WithInterval(20*time.Minute)).Do(func() (bool, error) {
```

Comment:
what will happen if schema changed during the refresh interval?

Reply:
Generally the schema is a stable resource, and the regular refresh basically meets the usage requirements. If there is a change during the interval, the request for the resource will fail